src/main/java/com/redhat/cloud/notifications/routers/models/SettingsValueJsonForm.java

```diff
@@ -29,16 +31,34 @@
         public List<Field> fields;
     }
 
-    public List<Field> fields = new ArrayList<>();
+    @JsonAutoDetect(fieldVisibility = Visibility.ANY)
+    public static class Sections extends Field {
+        public List<Field> sections = new ArrayList<>();
+
+        Sections() {
+            name = "notification-preferences";
+            component = COMPONENT_SECTION;
+        }
+    }
 
-    public static List<SettingsValueJsonForm> fromSettingsValue(SettingsValues values) {
-        ArrayList<SettingsValueJsonForm> forms = new ArrayList<>();
+    public List<Sections> fields = new ArrayList<>();
+
+    public static SettingsValueJsonForm fromSettingsValue(SettingsValues values) {
```

Comment:
This part of the code is a bit confusing for me but I suppose that's because I don't have a clear picture of what's going on in the UI. I don't know if that's possible but maybe a few comments would make it clearer.

Reply:
I know this is quite urgent so comments could also be added later.

Reply:
Thanks for the comments and link!